tools/wasi/CMakeLists.txt

```diff
@@ -7,8 +7,8 @@ include(ProjectUVWASI)
 add_library(wasi)
 add_library(fizzy::wasi ALIAS wasi)
 target_include_directories(wasi PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
-target_link_libraries(wasi PUBLIC fizzy::fizzy-internal PRIVATE uvwasi::uvwasi)
-target_sources(wasi PRIVATE wasi.cpp wasi.hpp)
+target_link_libraries(wasi PUBLIC fizzy::fizzy-internal uvwasi::uvwasi)
```

Comment:
Tests require now to include uvwasi's `<wasi_types.h>` to implement mock interface.

Is this ok change or should there be a better way?